addons/survey/tests/test_certification_flow.py

```diff
@@ -146,3 +148,76 @@ def test_flow_certificate(self):
         self.assertIn("employee@example.com", certification_email.email_to)
         self.assertEqual(len(certification_email.attachment_ids), 1)
         self.assertEqual(certification_email.attachment_ids[0].name, 'Certification Document.html')
+
+        user_inputs.unlink()
```

Comment:
I don't think you still need that part?

Reply:
You are totally right, my mistake :fearful: